Add Fabbot reusable workflow

nicolas-grekas opened the pull request.

nicolas-grekas committed:
Add Fabbot reusable workflow

wouterj commented:
😍 this has been on my idea list for way to long, great to see this getting traction!

nicolas-grekas commented:
I've reverted this because github doesn't supports reusable workflows in the special `.github` repository.
Moved to https://github.com/symfony/symfony/pull/61256 instead.
